common/utils/src/main/resources/error/error-conditions.json

```diff
@@ -3798,6 +3798,21 @@
     ],
     "sqlState" : "22003"
   },
+  "SYNTAX_DISCONTINUED" : {
+    "message" : [
+      "Support of the clause or keyword: <clause> has been discontinued in this context."
+    ],
+    "subClass" : {
+      "BANG_EQUALS_NOT" : {
+        "message" : [
+          "The '!' keyword is supported as a prefix operator in a logical operation only.",
```

Comment:
what is logical operation?

Reply:
NOT, AND, OR (?) I can rephrase, "Only supported for the prefix operator "NOT"?

Reply:
Rephrased